app/views/declaration.njk

```diff
@@ -40,7 +40,7 @@
       <li>you’re the registered keeper</li>
       <li>you’re applying for a review on a CPH registered in England</li>
       <li>{{ minNumText }}</li>
-      <li>you’ve carefully read and accept the <a href="/apply/terms">full terms and conditions</a></li>
+      <li>you’ve carefully read and accept the <a href="{{latestTermsAndConditionsUri}}" target="_blank">full terms and conditions</a></li>
```

Comment:
Same as previous comment, is this a new requirement to open in a new tab?

Reply:
removed as per https://design-system.service.gov.uk/styles/typography/#links:~:text=link%20icon.-,Opening%20links%20in%20a%20new%20tab,-Avoid%20opening%20links